requirements/lxml.txt

```diff
@@ -0,0 +1 @@
+lxml==4.5.0
```

Comment:
A few items:
a) I'm slightly confused why we're splitting `lxml` out into its own file!
b) `libweasyl/setup.py:L31` is declaring `'lxml==4.5.2',` is required, but we require `4.5.0` here?

Reply:
lxml takes longer to build than the rest of the dependencies combined! So, the Docker-based build [builds it in parallel](https://github.com/Weasyl/weasyl/blob/ec1e99954a846e0f921a30b7b5b64eee6572d3ab/Dockerfile#L22), which is where I got the version number. But apparently I merged some dependabot PRs without thinking about that. Thanks for finding the issue!